templates/admin/systemInfo.tpl

```diff
@@ -96,10 +96,24 @@
 						<td colspan="2" class="app--admin__systemInfoGroup">{$category}</td>
 					</tr>
 					{foreach from=$settings item="value" key="name"}
-						<tr>
-							<td>{$name|escape}</td>
-							<td>{$value|escape}</td>
-						</tr>
+						{if $category == "database"}
```

Comment:
Hi @Tyl13! I think this could be done a lot more succinctly with
```
{if $name == "password"}
	<td>********</td>
{else}
	<td>{$value|escape}</td>
{/if}
```
(Any setting named `password` should not be displayed.)

Reply:
I agree, I originally had it on all passwords that could be displayed. Also, that is a lot more succinct. I can't believe I didn't to limit it to the value td instead of the whole row. Thanks, I will fix and quickly throw back up.